src/platform/user/selectors.js

```diff
@@ -70,3 +82,37 @@ export const selectIsCernerOnlyPatient = state =>
 
 export const selectIsCernerPatient = state =>
   selectPatientFacilities(state)?.some(f => f.isCerner);
+
+// return the Cerner facilities that are _not_ blocked from Cerner's RX features
+export const selectCernerRxFacilities = state =>
+  selectPatientFacilities(state)?.filter(
+    f => f.isCerner && !CERNER_RX_BLOCKLIST.includes(f.facilityId),
+  );
+
+// return the Cerner facilities that are _not_ blocked from Cerner's secure
+// messaging features
+export const selectCernerMessagingFacilities = state =>
+  selectPatientFacilities(state)?.filter(
+    f => f.isCerner && !CERNER_MESSAGING_BLOCKLIST.includes(f.facilityId),
+  );
+
+// return the Cerner facilities that are _not_ blocked from Cerner's
+// appointments features
+export const selectCernerAppointmentsFacilities = state =>
```

Comment:
I'm trying to think through how VAOS would use these selectors, since I think we'd want to make use of the blocklist for appointments, to be consistent with the profile page. This selector wouldn't work for us directly, because we need to know the status of all the facilities, rather than filter down to the Cerner ones.

I wonder if it makes sense to have a selector that generates an array of objects like this:

```
{
    facilityId: '123',
    isCerner: true,
    usesCernerAppointments: true,
    usesCernerMedicalRecords: false,
    usesCernerMessaging: false,
    usesCernerRx: false,
    usesCernerTestResults: false,
}
```

VAOS could use that selector directly and these feature specific ones could use it underneath.

Reply:
I dig it. Will make that addition.

Reply:
Outside of making its internals a little more complex, is there any reason we can't just make `selectPatientFacilities()` _inflate_ the facilities and return objects like you describe?

Reply:
That sounds good to me. Seems reasonable to have this logic in there.

Reply:
Change made. Thanks for the suggestion.

Reply:
actually I need to revise what I put up...